cmd/tempo-vulture/vulture.go

```diff
@@ -0,0 +1,187 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/trace"
+	"go.uber.org/zap"
+)
+
+// Vulture is used to send traces to Tempo, and then read those traces back out to verify that the service is operating correctly.
+type Vulture struct {
+	completeChan     chan completeTrace
```

Comment:
the original design was to seed the random number generator with known values anchored on timestamps. this way vulture can query for any trace in the past 2 weeks and know exactly what it should look like without having to keep anything in memory. i'd prefer keeping that design unless there are strong reasons to do otherwise.

Reply:
It seemed like there was a race happening where we'd try to discover a past time stamp that was never sent and then check for it, and get a 404 back.  I like Vulture being stateless, in that nothing gets persisted between restarts.  Is there a middle ground here between trying to rediscover an old timestamp and keeping track of every trace id?  We could perhaps continue to query traces that had been sent in a sampled way if checking for older traces is important.  I definitely didn't factor that in.  Though, the long running traces should ensure that traces pull from multiple blocks.

Reply:
Generating and then validating a trace from a past timestamp is stateless :) which is one reason I like it so much. It's easy to validate a complete trace that was pushed last week without having to store it somewhere. I'd really like to keep it. 

I'm willing to endure a small number of 404s to keep this design. 

> We could perhaps continue to query traces that had been sent in a sampled way if checking for older traces is important.

Querying older traces is very important b/c it ensures that nothing changes as the trace is cut, flushed to the backend and repeatedly compacted over 2 weeks.